core/src/main/java/org/mule/runtime/core/internal/util/JarUtils.java

```diff
@@ -58,15 +58,20 @@ private JarUtils() {
 
```

Comment:
This JarUtils should be moved as down as possible in the dependency terr. It is internal, so it should be no problem to do that.

Reply:
We can move this whole class or parts of it (mostly staatic methods) to the new module